rclcpp/test/test_function_traits.cpp

```diff
@@ -142,15 +143,15 @@ TEST(TestFunctionTraits, arity) {
       return 0;
     };
 
-  auto lambda_one_int = [](int) {
+  auto lambda_one_int = [](int /*one*/) {
```

Comment:
I would suggest leaving out the variable name.
Only if an unused argument should be documented the body should contain something like this:

    (void)arg_name;